src/client/components/pages/parts/privs-edit-modal.js

```diff
@@ -93,6 +96,12 @@ class PrivsEditModal extends React.Component {
 		}
 	}
 
+	handleNoteChange(event) {
+		this.setState({
+			note: event.target.value
```

Comment:
Maybe adding `submittable: Boolean(event.target.value?.length)` here would be a good safeguard

Reply:
It will require some more conditions to work. I have made some changes, please see.